docusaurus/docs/how-to-guides/data-source-plugins/add-logs-metrics-traces-for-backend-plugins.md

```diff
@@ -1,6 +1,6 @@
 ---
-id: add-logs-metrics-traces-for-backend-plugins
-title: Add logs, metrics and traces for backend plugins
+id: add-logs-metrics-traces
+title: Add logs, metrics, and traces
```

Comment:
If changing the title we need to make it clearer on the page that this requires a backend component